src/styles/index.scss

```diff
@@ -1,3 +1,229 @@
+  }
+  &:nth-child(7)::after {
+    content: '7';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(8)::after {
+    content: '8';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(9)::after {
+    content: '9';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(10)::after {
+    content: '10';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(11)::after {
+    content: '11';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(12)::after {
+    content: '12';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(13)::after {
+    content: '13';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(14)::after {
+    content: '14';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(15)::after {
+    content: '15';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(16)::after {
+    content: '16';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(17)::after {
+    content: '17';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(18)::after {
+    content: '18';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(19)::after {
+    content: '19';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(20)::after {
+    content: '20';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(21)::after {
+    content: '21';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(22)::after {
+    content: '22';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(23)::after {
+    content: '23';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(24)::after {
+    content: '24';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(25)::after {
+    content: '25';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(26)::after {
+    content: '26';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(27)::after {
+    content: '27';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(28)::after {
+    content: '28';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(29)::after {
+    content: '29';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(30)::after {
+    content: '30';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+  &:nth-child(31)::after {
+    content: '31';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+}
+
+.calendar--month-length-28 :nth-child(n + 29),
+.calendar--month-length-29 :nth-child(n + 30),
+.calendar--month-length-30 :nth-child(n + 31),
+.calendar--month-length-31 :nth-child(n + 32) {
```

Comment:
The selectors for '.calendar--month-length-*' classes are missing a parent class or element that specifies which calendar they belong to. This could lead to unintended styling if there are multiple calendars on the page. Consider adding a more specific parent selector to avoid conflicts.